client/styles/abstracts/_variables.scss

```diff
@@ -44,6 +44,7 @@ $themes: (
     input-text-color: #333,
     input-border-color: #b5b5b5,
     about-list-text-color: #4a4a4a,
+    search-background-color: #ebebeb
```

Comment:
You should add this variable to the other themes, `dark` and `high-contrast`. For dark I would use the same color but for high contrast i would use `$white`.